ClientIdentifier.ts

```diff
@@ -0,0 +1,11 @@
+import { Identifier } from "authly"
+
+export namespace ClientIdentifier {
+
+	export function getClientId() {
```

Comment:
There is something of with the naming of things here. Maybe use a static class with at static property?